cmd/notary/tuf.go

```diff
@@ -249,9 +252,27 @@ func (t *tufCommander) tufAdd(cmd *cobra.Command, args []string) error {
 	if err = nRepo.AddTarget(target, t.roles...); err != nil {
 		return err
 	}
-	cmd.Printf(
-		"Addition of target \"%s\" to repository \"%s\" staged for next publish.\n",
-		targetName, gun)
+
+	cmd.Printf("Addition of target \"%s\" to repository \"%s\" staged for next publish.\n", targetName, gun)
+
+	// Auto publish the local trusted collection to remote.
+	if t.autoPublish {
+
+		// We need to set up a http RoundTripper when publishing
+		rt, err := getTransport(config, gun, readWrite)
+		if err != nil {
+			return err
+		}
+
+		nRepo, err := notaryclient.NewNotaryRepository(
+			config.GetString("trust_dir"), gun, getRemoteTrustServer(config), rt, t.retriever, trustPin)
+		if err != nil {
+			return err
+		}
+
```

Comment:
this looks good, the only thing I would add is to have a print similar to how we have one in `tufPublish`, something like:

```
...
cmd.Println("Auto-publishing changes to", gun)
return nRepo.Publish()
```